src/pages/students/Students.tsx

```diff
@@ -1,11 +1,192 @@
-import './Students.css'
+import "./Students.css";
+import { useForm } from "react-hook-form";
+import type { Resolver } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { studentsScheme } from "../students/students.scheme";
+import { MOCKED_PAYMENT_TYPES } from "../../mocks/paymentTypes";
+import { useNavigate } from "react-router-dom";
+import ErrorIcon from "../../assets/error-icon.png";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { Controller } from "react-hook-form";
 
+type FormData = {
+  name: string;
+  lastname: string;
+  nroTelefono: string;
+  fechaNacimiento: Date | null;
+  tipoPago: string;
+  cantidadDias: number;
+  diaPago: string;
+  diasExtra: string;
+  fechaIngreso: Date | null;
+};
 function Students() {
+  const navigate = useNavigate();
+  const resolver = yupResolver(studentsScheme) as Resolver<FormData>;
+
+  const onSubmit = (data: any) => {
+    console.log("Formulario enviado con datos:", data);
+  };
+  const {
+    register,
+    handleSubmit,
+    watch,
+    control,
+    formState: { errors },
+  } = useForm<FormData>({
+    resolver,
+    defaultValues: {
+      name: "",
+      lastname: "",
+      nroTelefono: "",
+      fechaNacimiento: null,
+      tipoPago: "",
+      diaPago: "",
+      diasExtra: "",
+      fechaIngreso: null,
```

Comment:
No hace falta que definas que el valor por defecto de los tipos de datos String van a ser un string vacio, ya que por defecto lo son. Con todos los cambios te deberia quedar asi

```suggestion
      fechaNacimiento: new Date(),
      fechaIngreso: new Date()
```

Reply:
Ahí probé pero el tema es que no se entiende que fecha es que:
![image](https://github.com/user-attachments/assets/12909048-10fe-40c8-ad7c-e9b75dad7d83)
Le agrego un label a esos dos inputs o habrá otra forma?

Reply:
Sip, agrega los labels a todos los inputs ya que los placeholders no se utilizan para describir qué es el campo sino que es una ayuda para indicar cómo se debe de llenar el campo